### ISSUE TITLE
Add Monarch App backlinks to Mondo IDs

### ISSUE THREAD

**Issue by cmungall** [monarch-initiative/mondo#1217]
OLS or Monarch site? I think the former as the latter will lag. cc @mellybelly

**Comment by matentzn** [comment]
You mean the default redirect of term IRIs? Like http://purl.obolibrary.org/obo/MONDO_0000001

**Comment by cmungall** [comment]
correct, the OBO PURLs. We also want to make this prominent in user documentation, to guide people implementing external infrastructure who want to link to MONDO

It would be good to have a linkout mechanism in OLS to link to Monarch from there

**Comment by matentzn** [comment]
How would the backlink work? Via xref or something else that hardcodes a link to monarch initiative?

**Comment by cmungall** [comment]
identifiers.org used to give a choice of sites, now it seems to go to a default

there could be a registry of prefixes with expansions. This could be provided in OBO metadata

Or it could be a list of IDs that are submitted to OLS. Like NCBI linkouts

**Comment by matentzn** [comment]
Need to be discussed in meeting.

**Comment by joeflack4** [comment]
@matentzn Agreed. Sorry if you guys are having to discuss these issues a second time w/ Dazhi gone. I've added `needs clarification` label to this. I have some idea as to how I would do this in OMIM, but I'm not sure where I would go within this codebase to do this work. As for making URLs resolving (if or how), I'm not too sure on that either.

**Comment by sabrinatoro** [comment]
This issue was created more than 2 years ago and has not been addressed to date. It has therefore been identified as a candidate to be closed without action.
If anyone believes that this issue should be addressed, please comment here.
Otherwise, this issue will automatically be closed with the October 2025 release.

Note: @matentzn I think this is already done, and we can close this issue. Thoughts?

**Comment by matentzn** [comment]
These exist for a long time

![Image](https://github.com/user-attachments/assets/3d60d2dc-90db-4ccc-8c5c-2e0fd915f0e5)
